lib/modules/manager/gradle/artifacts.ts

```diff
@@ -152,7 +152,10 @@ export async function updateArtifacts({
   }
 
   const gradlewName = gradleWrapperFileName();
-  const gradlewFile = await findUpLocal(gradlewName, dirname(packageFileName));
+  const gradlewFile = await findUpLocal(
+    gradlewName.replace(new RegExp(`^\\./`), ''),
```

Comment:
```suggestion
    gradlewName.replace(new regEx(`^\\./`), ''),
```
use our regex helper